test/k8s/simple_test.go

```diff
@@ -6,30 +6,21 @@ import (
 	"fmt"
 	"os"
 	"strconv"
-	"strings"
 	"sync"
 	"time"
 
-	"github.com/gruntwork-io/terratest/modules/random"
-	. "github.com/kong/mesh-perf/test/framework"
 	"github.com/kumahq/kuma-tools/graph"
 	"github.com/kumahq/kuma/pkg/config/core"
 	. "github.com/kumahq/kuma/test/framework"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+
+	. "github.com/kong/mesh-perf/test/framework"
 )
 
 func Simple() {
 	BeforeAll(func() {
-		opts := []KumaDeploymentOption{
-			WithInstallationMode(HelmInstallationMode),
-			WithHelmChartVersion(meshVersion),
-			WithHelmReleaseName(fmt.Sprintf("kuma-%s", strings.ToLower(random.UniqueId()))),
-			WithHelmChartPath(Config.HelmChartPath), // we pass chart name to use production chart
-			WithoutHelmOpt("global.image.tag"),      // required to use production chart
-			WithHelmOpt(`controlPlane.podAnnotations.prometheus\.io/port`, "5680"),
-			WithHelmOpt(`controlPlane.podAnnotations.prometheus\.io/scrape`, "true"),
-		}
```

Comment:
Could you help me understand why it's not needed anymore?

Is it because we now use kumactl to install and we fetch it for the test?

Reply:
Are you sure we needed these annotations in the first place? There are annotations on the CP Service:
```yaml
apiVersion: v1
kind: Service
metadata:
  name: kong-mesh-control-plane
  namespace: kong-mesh-system
  labels:
    app: kong-mesh-control-plane
    app.kubernetes.io/name: kong-mesh
    app.kubernetes.io/instance: kong-mesh
  annotations:
    prometheus.io/scrape: "true"
    prometheus.io/port: "5680"
spec:
  type: ClusterIP
  ports:
    - port: 5680
      name: diagnostics
      appProtocol: http
    - port: 5681
      name: http-api-server
      appProtocol: http
    - port: 5682
      name: https-api-server
      appProtocol: https
    - port: 443
      name: https-admission-server
      targetPort: 5443
      appProtocol: https
    - port: 5676
      name: mads-server
      appProtocol: https
    - port: 5678
      name: dp-server
      appProtocol: https
  selector:
    app: kong-mesh-control-plane
    app.kubernetes.io/name: kong-mesh
    app.kubernetes.io/instance: kong-mesh
```
and it seems like they're sufficient for scraping to work

Reply:
Annotations are one thing, but my question was also regarding other options that we set for the test.

Annotations on pod might be redundant. This might have been a mistake on my side. If it works, then we can get rid of them

Reply:
Yes, we're using fetched `kumactl` of the specific version to install KM. I think it's the easiest way to test arbitrary KM commits. I'll update PR's description